src/tests/suite/daos_rebuild.c

```diff
@@ -1018,7 +1018,7 @@ rebuild_master_failure(void **state)
 	pinfo.pi_bits = DPI_REBUILD_STATUS;
 	rc = test_pool_get_info(arg, &pinfo, &affected_engines);
 	assert_rc_equal(rc, 0);
-	assert_int_equal(pinfo.pi_rebuild_st.rs_done, 1);
+	assert_int_equal(pinfo.pi_rebuild_st.rs_state, 2);
```

Comment:
[Question] Is this fixing an existing bug that is unrelated to our changes? Just curious how you found it.

Reply:
I had to merge master due to a merge conflict in this file. The change was made in PR https://github.com/daos-stack/daos/pull/7885